src/Connector.sol

```diff
@@ -138,9 +138,14 @@ contract CentrifugeConnector {
         public
         onlyRouter
     {
+        // Lookup the tranche token
```

Comment:
I have a comment about Solidity coding style ;-) I'll try to find some time to write down my personal views on this soon, so we can align on this and see if we agree as a team. Because most important is that we are consistent on this, but I can see that's hard since we've never documented this (afaik).

But for now, I think we should follow the MakerDAO Solidity coding patterns. In terms of comments, this means optionally adding natspec doc comments to the functions, but only adding inline comments if code is unclear.

For me the guideline I would describe is: if the comment line is basically saying the same thing as the line of code, the comment should go. E.g. looking at this:
```solidity
// Mint the transfer amount to the destinationAddress account
token.mint(destinationAddress, amount);
```

The code itself is very explicit about what it's doing, the comment doesn't add anything.

For reference, here's some Maker D3M code again: https://github.com/makerdao/dss-direct-deposit/blob/master/src/D3MHub.sol#L366 You can see there's no comments for most lines that are clear enough already, but there are comments for lines that are less clear.

Personally, I think this code is more readable: simple lines of code that are explicit and use expressive naming to not need comments, with blocks of code that are more complex still having comments to describe their behaviour.

Reply:
I agree 100% with you and this guy wrote something about this almost 7 years ago actually: https://nunoalexandre.com/2016/04/29/comments 😛 

I just added the comments for now for me to separate the different things happening within the function to more easily make sense of it. Happy to drop it all before merging 👍